ui/app/models/replication-attributes.js

```diff
@@ -28,6 +28,16 @@ export default Fragment.extend({
       ? 'bootstrapping'
       : (this.get('isSecondary') && 'secondary') || (this.get('isPrimary') && 'primary');
   }),
+  modeForHeader: computed('mode', function() {
```

Comment:
This separates out a specific property for the mode displayed in the Header.  We needed one that wasn't also moonlighting for a property from the URL (e.g. modeForUrl). 
![image](https://user-images.githubusercontent.com/6618863/82491510-f63fa600-9aa1-11ea-92ac-98aa5012eb41.png)